.github/workflows/tests-smoke-ipv6.yaml

```diff
@@ -114,23 +120,36 @@ jobs:
           # To make sure that cilium CRD is available (default timeout is 5m)
           kubectl wait --for condition=Established crd/ciliumnetworkpolicies.cilium.io --timeout=5m
 
+          kubectl port-forward -n kube-system deployment/hubble-relay 4245:4245 &
+
       - name: Run conformance test (e.g. connectivity check without external 1.1.1.1 and www.google.com)
         run: |
           kubectl apply -f ${{ env.CONFORMANCE_TEMPLATE }}
           kubectl wait --for=condition=Available --all deployment --timeout=${{ env.TIMEOUT }}
 
       - name: Capture cilium-sysdump
         if: ${{ failure() }}
+        # The following is needed to prevent hubble from receiving an empty
+        # file (EOF) on stdin and displaying no flows.
+        shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
```

Comment:
This was the change that was required to resolve https://github.com/cilium/cilium/pull/16968#issuecomment-885964109.

Reply:
:exploding_head: